SLC-S-GQIMonitor/SLC-AS-GQIMonitor.VersionCheck/SLC-AS-GQIMonitor.VersionCheck.cs

```diff
@@ -60,7 +55,12 @@ private void RunSafe(IEngine engine)
 				return;
 
 			var requiredVersion = requiredVersionParam.Value;
-			var actualVersion = GetGQIVersion(engine.GetUserConnection());
+			var actualVersion = GetGQIVersion(engine.GetUserConnection(), out var featureInfo);
+
+			if (string.IsNullOrEmpty(actualVersion))
+			{
+				return;
```

Comment:
I would expect a popup if the actualVersion could not be resolved?

Reply:
The user knows about this version check when the version is not supported.

The popup shouldn't say anything technical about an API call or exception of course but should probably still indicate that the page might not be supported since we don't know if they are on a supported version.